content/en/docs/tasks/configure-pod-container/assign-pod-level-resources.md

```diff
@@ -0,0 +1,269 @@
+---
+title: Assign Pod-level CPU and memory resources
+content_type: task
+weight: 30
+min-kubernetes-server-version: 1.32
+---
+
+
+<!-- overview -->
+
+{{< feature-state feature_gate_name="PodLevelResources" >}}
+
+This page shows how to specify CPU and memory resources for a Pod at pod-level in
+addition to container-level resource specifications. A Kubernetes node allocates
+resources to a pod based on the pod's resource requests. These requests can be
+defined at the pod level or individually for containers within the pod. When
+both are present, the pod-level requests take precedence.
+
+Similarly, a pod's resource usage is restricted by limits, which can also be set at
+the pod-level or individually for containers within the pod. Again,
+pod-level limits are prioritized when both are present. This allows for flexible
+resource management, enabling you to control resource allocation at both the pod and
+container levels.
+
+In order to specify the resources at pod-level, it is required to enable
+`PodLevelResources` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/).
+
+For Pod Level Resources:
+* Priority: When both pod-level and container-level resources are specified,
+  pod-level resources take precedence.
+* QoS: Pod-level resources take precedence in influencing the QoS class of the pod.
+* OOM Score: The OOM score adjustment calculation considers both pod-level and
+  container-level resources.
+* Compatibility: Pod-level resources are designed to be compatible with existing
+  features.
+
+## {{% heading "prerequisites" %}}
+
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+The `PodLevelResources` [feature
+gate](/docs/reference/command-line-tools-reference/feature-gates/) must be enabled
+for your control plane and for all nodes in your cluster.
+
+## Create a pod with memory requests and limits at pod-level
+
+To specify resources requests for a Pod at pod-level, include the resources:requests
+field in the Pod spec manifest. To specify a memory limit, include resources:limits.
```

Comment:
just memory limit?

Reply:
oh I see. I find it weird that the first sentence is resource agnostic "to specify resource requests" vs the limits are specifically about memory

Reply:
Fixed the sentence, please check if it looks better now

Reply:
yup looks better! thanks